src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.video.unit.spec.jsx

```diff
@@ -231,9 +234,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
 
       fireEvent.click(await screen.findByText(/Details/));
 
-      await waitFor(() =>
-        expect(screen.history.push.lastCall.args[0]).to.equal(url),
-      );
+      await screen.findByText(/Cheyenne VA medical center/i);
```

Comment:
What happens with these tests when we go from the list to the details page is:

1. List page is shown
2. Details link is clicked before facility data loads
3. Details page is shown with content that doesn't require facility info
4. Details page action sees that there's no facility info, puts up a spinner while fetching
5. Details page content is shown again when facility fetch is done

This is kind of a pain, since the content is there, then disappears. Waiting for the facility name to display waits until all the loading is done.

This is arguably an actual bug, but there are a couple of future changes that will make it irrelevant, which is that we're going to be required to have the facility info before showing anything on the list page and that we're doing to include the facility info in the appointments fetch in the v2 endpoints. Either of those will change how the list page works and avoid this bug.